textual/src/toga_textual/window.py

```diff
@@ -1,17 +1,96 @@
+from rich.text import Text
+
+from textual.app import RenderResult
+from textual.reactive import Reactive
 from textual.screen import Screen as TextualScreen
-from textual.widgets import Header as TextualHeader
+from textual.widget import Widget as TextualWidget
 
 from .container import Container
 
 
+class CloseIcon(TextualWidget):
+    DEFAULT_CSS = """
+    CloseIcon {
```

Comment:
In the [standard Textual header](https://github.com/Textualize/textual/blob/v0.34.0/src/textual/widgets/_header.py), I don't think this was intended to be a close button, but an application icon, as it's in the place where the icon would be on Windows. 

Now that I'm a Mac user, I thought it was a close button too, and spent some time trying to work out why it didn't do anything. To avoid this confusion, maybe we'd be better off just removing it.

Reply:
That's a fair point - it's a decoration, but not a functional one. 

I guess the alternative would be to *make* it functional... turn it into a button that does the close action on the window...